packages/grafana-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -73,7 +73,14 @@ export const Dropdown = React.memo(({ children, overlay, placement, offset, onVi
   const animationDuration = 150;
   const animationStyles = useStyles2(getStyles, animationDuration);
 
-  const onOverlayClicked = () => {
+  const onOverlayClicked = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    if (event.target instanceof HTMLDivElement || event.target instanceof HTMLSpanElement) {
+      const innerText = event.target.innerText
+      if (innerText.includes("More...")) {
+        return
+      }
+    }
+
```

Comment:
this is definitely too brittle i'm afraid 😬

Reply:
thanks for your review, bro. i will modify it.